packages/slots/components/MainFooter.tsx

```diff
@@ -0,0 +1,38 @@
+import { SlotComponentProps } from '../SlotRenderer';
+import { Container, Link } from '@plone/components';
+import Logo from './Logo';
+
+const Footer = (props: SlotComponentProps) => {
+  const { content, location } = props;
+  const siteActions = content?.['@components']?.actions?.site_actions || [];
+
+  return (
+    <Container layout className="footer">
+      {/* TODO: i18n properly */}
+      <div className="footer-message">
+        The Plone® Open Source CMS/WCM is © 2000-2024 by the Plone Foundation
+        and friends. <br />
```

Comment:
We should be explicit or remove this statement. I thought the Zope Foundation ceased to exist for the purpose of Plone, so we have no friends. so lonely...

Reply:
Guillotina? Zope people still exist even if their foundation doesn’t 😆

Reply:
It should go out, at least in here and only shown in demo instances, for example.